docs/model/custom-entities.md

```diff
@@ -4,7 +4,7 @@ When you need to add functionality that is not in the system, like an advertisin
 then you need your own, custom entities.
 
 * The system is prepared and configured for custom entities.
-The configuration is placed in [doctrine.yml](/project-base/app/config/packages/doctrine.yml), section `doctrine.orm.mappings`.
+The configuration is placed in [doctrine.yml](/config/packages/doctrine.yml), section `doctrine.orm.mappings`.
```

Comment:
Why did you change this path? It 404's now.

Reply:
fixed